sdk/python/feast/feature_store.py

```diff
@@ -733,3 +734,15 @@ def _print_materialization_log(
             f" to {Style.BRIGHT + Fore.GREEN}{end_date.replace(microsecond=0).astimezone()}{Style.RESET_ALL}"
             f" into the {Style.BRIGHT + Fore.GREEN}{online_store}{Style.RESET_ALL} online store.\n"
         )
+
+
+def _validate_feature_views(feature_views: List[FeatureView]):
+    """ Verify feature views have unique names"""
+    name_to_fv_dict = {}
+    for fv in feature_views:
+        if fv.name in name_to_fv_dict:
+            raise ValueError(
+                f"More than one feature view with name {fv.name} found. Please ensure that all feature view names are unique."
```

Comment:
A common reason for this exception is that users have test scripts inside of their repository. What if we made this exception 

```suggestion
                f"More than one feature view with name {fv.name} found. Please ensure that all feature view names are unique. It may be necessary to ignore certain files in your feature repository by using a .feastignore file."
```

Reply:
Make sense. I added additional test case to test multiple scripts containing same feature view names inside a repo.